stdlib/pushbullet/pushbullet.flux

```diff
@@ -0,0 +1,73 @@
+package pushbullet
+
+import "http"
+import "json"
+
+option defaultURL = "https://api.pushbullet.com/v2/pushes"
+
+// `pushData` sends a push notification using PushBullet's APIs.
+// `url` - string - URL of the PushBullet endpoint. Defaults to: "https://api.pushbullet.com/v2/pushes".
+// `token` - string - the api token string.  Defaults to: "".
+// `data` - object - The data to send to the endpoint. It will be encoded in JSON and sent to PushBullet's endpoint.
```

Comment:
Cool idea. I'm not familiar with how we decide what notifications are useful. Are you planning to add any limitations or guidance on what data should be sent?

Reply:
Like do we already have certain conventions around notifications in the web UI or something?

Reply:
Hey, I think this is more related to PushBullet APIs.

It's hard to find a standardized way of accessing it.

So I decided to offer a "deafult" option via the `endpoint` function.

And a `genericEndpoint` for the ones who want to specify the entire payload (auth is managed for them).

Reply:
Ah, yeah thanks for the `slack`/`pagerduty` comparison. Helped me understand the convention a little bit better. I also guess that you're adding this as a vehicle for alerts, which makes more sense to me than a generic notification sytem.